### PR TITLE
Add Upgradeability Report components

### PR DESCRIPTION
**Pull request by sdlyy**
Resolves L2B-1853

### CODE REVIEW

File: packages/frontend/src/components/announcement/logos/AnnouncementLogos.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+
+export function L2BeatLogo() {
```
Comment (sz-piotr) [inline packages/frontend/src/components/announcement/logos/AnnouncementLogos.tsx:3]: We already have our logo in the codebase

**Review by sz-piotr — CHANGES_REQUESTED** [review]

File: packages/frontend/src/pages/scaling-tvl/view/ScalingTvlPage.tsx [hunk 2]
```diff
@@ -43,6 +45,7 @@ export function ScalingTvlPage(props: TvlPageProps) {
           />
           <ScalingTvlView {...props.tvlView} />
           <OtherSites />
+          {props.showMultisigReportBanner && <FullAnnouncementBanner />}
```
[the remark this answers is not in the document]
Comment (sdlyy) [reply packages/frontend/src/pages/scaling-tvl/view/ScalingTvlPage.tsx]: It should
https://github.com/l2beat/l2beat/pull/1692/commits/f6dcfb2724a450183a1fee7ae56de7ad5c3d43b0

File: packages/frontend/src/pages/meta-images/index.tsx [hunk 3]
```diff
@@ -28,23 +30,35 @@ export function getMetaImagePages(
     {
       slug: '/meta-images/overview-scaling',
       page: (
-        <PageWrapper {...scaling.wrapper}>
+        <PageWrapper
+          {...scaling.wrapper}
+          hideAnnouncementBar={shouldHideMultisigAnnouncement}
+          hideFloatingBanner={shouldHideMultisigAnnouncement}
```
[the remark this answers is not in the document]
Comment (sdlyy) [reply packages/frontend/src/pages/meta-images/index.tsx]: https://github.com/l2beat/l2beat/pull/1692/commits/9395e15d2362fc99d811c5a940bf77205bee6d14

File: packages/frontend/src/components/navbar/types.ts [hunk 4]
```diff
@@ -4,3 +4,4 @@ export type NavbarPage =
   | 'donate'
   | 'faq'
   | 'definitions'
+  | 'multisig-report'
```
[the remark this answers is not in the document]
Comment (sdlyy) [reply packages/frontend/src/components/navbar/types.ts:7]: https://github.com/l2beat/l2beat/pull/1692/commits/9ef853bdfae9f4dd349dc508090f1b41deaae7fa

File: packages/frontend/src/components/navbar/SidebarMenu.tsx [hunk 5]
```diff
@@ -85,6 +87,11 @@ export function SidebarMenu(props: SidebarMenuProps) {
           <li>
             <a href="/donate">Donate</a>
           </li>
+          {props.showMultisigReport && (
```
[the remark this answers is not in the document]
Comment (sdlyy) [reply packages/frontend/src/components/navbar/SidebarMenu.tsx]: https://github.com/l2beat/l2beat/pull/1692/commits/9ef853bdfae9f4dd349dc508090f1b41deaae7fa

File: packages/frontend/src/components/navbar/types.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (sdlyy) [reply packages/frontend/src/components/navbar/types.ts:7]: https://github.com/l2beat/l2beat/pull/1692/commits/9ef853bdfae9f4dd349dc508090f1b41deaae7fa

File: packages/frontend/src/components/announcement/logos/AnnouncementLogos.tsx (hunk 1 above)
Comment (sdlyy) [reply packages/frontend/src/components/announcement/logos/AnnouncementLogos.tsx]: https://github.com/l2beat/l2beat/pull/1692/commits/4640c83ac2cfcabff5aa32479a69f25ce45c2447

File: packages/frontend/src/components/announcement/logos/AnnouncementLogos.tsx [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+      <defs>
+        <clipPath id="clip0_803_46771">
+          <rect width="63.7" height="26" fill="white" />
+        </clipPath>
+      </defs>
+    </svg>
+  )
+}
+
+export function PolygonLogo() {
```
[the remark this answers is not in the document]
Comment (sdlyy) [reply packages/frontend/src/components/announcement/logos/AnnouncementLogos.tsx:3]: Polygon logo also handles darkmode fill transformation here, so I believe it's easier to stick with the SVG

File: packages/frontend/src/components/announcement/AnnouncementBanner.tsx [hunk 7]
```diff
@@ -0,0 +1,111 @@
+import cx from 'classnames'
+import React from 'react'
+
+import { AnnouncementButtonLink } from './AnnouncementButtonLink'
+import { AnnouncementImage } from './AnnouncementImage'
+import { AnnouncementNotation } from './AnnouncementNotation'
+import { AnnouncementTitle } from './AnnouncementTitle'
+import { L2BeatLogo, PolygonLogo } from './logos/AnnouncementLogos'
+
+export function FullAnnouncementBanner() {
```
[the remark this answers is not in the document]
Comment (sdlyy) [reply packages/frontend/src/components/announcement/AnnouncementBanner.tsx:6]: ty for insights 🙏🏻 
https://github.com/l2beat/l2beat/pull/1692/commits/8a407b709e513e389067b0b2892edf998dcf6a51

File: packages/frontend/src/pages/bridges-projects/index.tsx [hunk 8]
```diff
@@ -19,7 +21,11 @@ export function getBridgeProjectPages(config: Config, pagesData: PagesData) {
     return {
       slug: `/bridges/projects/${bridge.display.slug}`,
       page: (
-        <PageWrapper {...wrapper}>
+        <PageWrapper
+          {...wrapper}
+          hideAnnouncementBar={shouldHideMultisigAnnouncement}
+          hideFloatingBanner={shouldHideMultisigAnnouncement}
```
[the remark this answers is not in the document]
Comment (sdlyy) [reply packages/frontend/src/pages/bridges-projects/index.tsx]: https://github.com/l2beat/l2beat/pull/1692/commits/883e97d0c1053b495bf4e244fb3b3eb535a2ebb0

**Review by sz-piotr — APPROVED** [review]

**Pull request merged**